src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('li');
+
+function sortList(list) {
+  list.sort((person1, person2) => person2.salary - person1.salary);
```

Comment:
The `sortList` function expects a list of employee objects, but it is being passed the result of `getEmployees`, which is an array of employee objects. This is correct, but ensure that the `list` parameter in `sortList` is indeed an array of objects with `salary` properties.